dataland-frontend/tests/e2e/specs/low/general/Authorisation.ts

```diff
@@ -14,4 +14,13 @@ describe("Authentication Buttons", () => {
     cy.login();
     cy.logoutDropdown();
   });
+
+  it("Checks that the back button works as expected", () => {
```

Comment:
Vielleicht ergänzen: Checks that the back button on the login page works as expected.
Da der Backbutton dort ja eine andere Funktionalität besitzt als an anderen Stellen der Seite.

Reply:
OK. Done

Reply:
Ok, passt.